SWLOR.Game.Server/Feature/PerkDefinition/OneHandedPerkDefinition.cs

```diff
@@ -103,19 +103,19 @@ private void DualWield(PerkBuilder builder)
                 .Name("Dual Wield")
 
                 .AddPerkLevel()
-                .Description("Enables the use of two one-handed weapons at the same time at standard NWN penalties.")
+                .Description("Enables the use of two one-handed weapons at the same time at -30%/-50% to hit.")
```

Comment:
I went ahead and updated the spreadsheets to include these changes. I'm using that as our bible so if we end up making mechanical changes to perks, please try to keep the spreadsheet in sync too.

Reply:
(We'll need to change this again based on our conversation).

Got it - this was really a cosmetic change because the function is the same, I was just mapping the text over to the new mechanics for clarity.  But no reason the same logic doesn't apply to the s/s!